Rubberduck.VBEEditor/Utility/FileExistenceChecker.cs

```diff
@@ -9,9 +10,16 @@ public interface IFileExistenceChecker
 
     public class FileExistenceChecker : IFileExistenceChecker
```

Comment:
I was wondering why `FileExistenceChecker` needed to exist with `FileSystem` available.  So, I poked around - and, near as I can tell, the `FieldExistenceChecker` constructor is never referenced.  Can it be deleted?  Even if it is used in some way, could it simply be replaced by `FileSystem`?

Reply:
I agree that `FileExistenceChecker` is now superficial but I originally decided to not do it as I didn't want to cause more changes and potentially problems. However I'll double check whether it's in fact referenced. It definitely does need to be replaced with the `IFileSystem`.

Reply:
Ding dong! `FileExistenceChecker` is dead. Long live `FileSystem`!